source/rc.inetd/git/env/CMD

```diff
@@ -0,0 +1 @@
+git
```

Comment:
Will there be a problem if we invoke `/usr/libexec/git-core/git-daemon` directly? Because it seems that `git daemon` will just invoke `/usr/libexec/git-core/git-daemon` with all command line arguments given after `git daemon` passed to `/usr/libexec/git-core/git-daemon`. Invoking `/usr/libexec/git-core/git-daemon` directly will simplify the process tree as shown in htop tree format (for example).